### ISSUE TITLE
Add approval status to PR cards

### ISSUE THREAD

**Issue by matt-carr** [integrations/microsoft-teams#109]
**Describe the solution you'd like**
It would be really handy if the PR cards had a section for showing approvals, similar to checks.

e.g.

*Reviewers:* @joe-schmoe, @matt-carr ✔️, @someone-else ❌, @a-third-person 💬

**Describe alternatives you've considered**
Alternatively, simply a status that shows whether or not a PR is ready to merge

e.g.

*Reviewers:* @joe-schmoe, @matt-carr, @someone-else, @a-third-person
*Review Status:* Changes Requested ❌ 
*Review Status:* Approved ✔️ 
*Review Status:* Open

**Comment by matt-carr** [comment]
If there's further discussion after a review, the reviews may be buried in the comments thread - it would be ideal to have all the critical PR information available at a glance in the main card

**Comment by bradcutshall** [comment]
@ashokirla  @matt-carr I agree. Just recently posted a reply to this thread with a similar request:
https://github.com/integrations/microsoft-teams/issues/45#issuecomment-1286146027
